bin/rapptor.js

```diff
@@ -4,17 +4,33 @@ const Rapptor = require('../');
 const path = require('path');
 
 let cwd = process.cwd();
+let justConfig = false;
```

Comment:
lets remove the cwd override logic - we've never used it.  that'll clean up the logic a bit